LevelUpReward.lua

```diff
@@ -37,17 +39,26 @@ Config_ItemAmount[35] = 10
 Config_ItemId[39] = 9312
 Config_ItemAmount[39] = 5
 
-local Config_mailText = 2
-local senderGUID = 1
+-- General Settings Config
+local Config_mailText = 1           -- Which text to send in the mail to the player.
+local Config_senderGUID = 1         -- GUID of the player found in CharactersSchema/Characters in Database.
+local Config_mailStationery = 61    -- Stationary used in the mail sent to the player. (41 Normal Mail, 61 GM/Blizzard Support, 62 Auction, 64 Valentines, 65 Christmas) Note: Use 62, 64, and 65 At your own risk.
```

Comment:
same as above, must be 41 for the "Create a permanent copy of the letter" to work